packages/module/src/components/AccessConsoles/AccessConsoles.tsx

```diff
@@ -135,9 +150,9 @@ export const AccessConsoles: React.FunctionComponent<AccessConsolesProps> = ({
     }
   });
   return (
-    <div className={css(styles.console)}>
+    <div className={clsx(styles.console)}>
```

Comment:
```suggestion
    <div className={styles.console}>
```
I think there is no need to use clsx in such cases